back/courses/templates/courses/add_courses.html

```diff
@@ -0,0 +1,62 @@
+{% load static %}
+<!--Header-->
+{% include "header.html" with title="Ajout d'un cours" %}
+
+<!--Body-->
+<body>
+    <!--Feedback Button-->
+    {% include "feedback.html" %}
+
+    <!--Sidenav-->
+    {% include "sidenav.html" with active_link="admin" %}
+
+    <!--Content-->
+    <div class="content">
+        <div class="content-header">
+            <div class="row justify-content-between w-100">
+                <div class="col">
```

Comment:
Delete the search bar (it is useless here)

Reply:
Done